drivers/xaudio2/SCsub

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+
+Import('env')
+
+env.add_source_files(env.drivers_sources, "*.cpp")
+env.Append(CXXFLAGS=['-DXAUDIO2_ENABLED'])
+env.Append(LINKFLAGS=['xaudio2_8.lib'])
```

Comment:
In `detect.py` it is eventually set in the `LINKFLAGS` anyway: `env.Append(LINKFLAGS=[p+".lib" for p in LIBS])`. For Windows I can see no other way, since adding to SCons LIBS doesn't work.